devops/continuous_integration/packages_pipeline/multi_configuration.rst

```diff
@@ -0,0 +1,178 @@
+Package pipeline: multi configuration
+=====================================
+
+In the previous section we were building just 1 configuration. This section will cover the case in which we need to build more
+than 1 configuration. We will use the ``Release`` and ``Debug`` configurations here for convenience, as it is easier to 
+follow, but in real case these configurations will be more like Windows, Linux, OSX, building for different architectures, 
+cross building, etc.
+
+Let's begin cleaning our cache and initializing only the ``develop`` repo:
+
+
+.. code-block:: bash
+
+    $ conan remove "*" -c  # Make sure no packages from last run
+    $ conan remote remove "*"  # Make sure no other remotes defined
+    $ conan remote add develop <url-develop-repo>  # Add only the develop repo
+
+
+We will create the packages for the 2 configurations sequentially in our computer, but note these will typically run
+in different computers, so it is typical for CI systems to launch the builds of different configurations in parallel.
+
+.. code-block:: bash
+    :caption: Release build
+
+    $ conan create . --build="missing:ai/*" -s build_type=Release --format=json > graph.json
+    $ conan list --graph=graph.json --graph-binaries=build --format=json > upload_release.json
+    $ conan remote add packages "<url-packages-repo>"
+    $ conan upload -l=upload_release.json -r=packages -c --format=json > upload_release.json
```

Comment:
```suggestion
    $ conan list --graph=graph.json --graph-binaries=build --format=json > upload.json
    $ conan remote add packages "<url-packages-repo>"
    $ conan upload -l=upload.json -r=packages -c --format=json > upload_release.json
```
Using the same name makes it fail because the file is deleted for output before it's read by Conan